src/main/java/io/reactivex/exceptions/OnErrorNotImplementedException.java

```diff
@@ -47,7 +47,7 @@ public OnErrorNotImplementedException(String message, Throwable e) {
      * @param e
      *          the {@code Throwable} to signal; if null, a NullPointerException is constructed
      */
-    public OnErrorNotImplementedException(Throwable e) {
+    public OnErrorNotImplementedException(@Nullable Throwable e) {
```

Comment:
The argument should be non-null. It's just a safeguard to make sure a null is recorded as NPE.